frb/galaxies/photom.py

```diff
@@ -63,4 +66,18 @@ def merge_photom_tables(new_tbl, old_file, tol=1*units.arcsec, debug=False):
     return merge_tbl
 
 
+def extinction_correction(filter_name, EBV=0.138, RV=3.1):
```

Comment:
Please add a brief doc string including definition of the parameters (like most other methods in the Repo).